src/content/1.7/contribute/contribution-guidelines/_index.md

```diff
@@ -173,18 +173,31 @@ It is possible to add new dependencies, after a careful selection and study. A f
 
 ### Licenses
 
-The first step is to check the **legal compliance** of a third party software. The PrestaShop open source project being distributed under the OSL license, dependencies must be compatible with it. 
+The first step is to check the **legal compliance** of a third party software. The PrestaShop open source project being written and distributed under the OSL license, aggregated dependencies must be compatible with it. 
 
-Below is a list of the known compatible licenses that can accepted:
+Below is a list of the known compatible licenses that can accepted for software:
 
 - MIT
 - ISC
 - BSD
 - AFL
 - EUPL
-- Apache 
+- Apache
+- CC-O
+- Unlicense
 
-Then, a last check is done to be sure that there is no additional clause that would introduce a restriction of use.
+Additionally, artwork (e.g: icons, pictures, fonts, but not only) is usually distributed with specific licenses. Below is a list of known compatible licenses that can be accepted for artwork:
+
+- CC-0
+- CC-by-sa
+- CC-by
+- Art Libre
+- Artistic
+- Unlicense
+
+In general, public domain is not acceptable, as it doesn't exist in all juridictions or countries.
```

Comment:
Are you certain about this? We are currently using these flags for example, which don't specify one of the aforementioned licenses: http://www.famfamfam.com/lab/icons/flags/

Reply:
certain about what ? artistic / unlicense? or public domain? or both?

Reply:
I checked famfamfam icons pack. There is a readme that give more information: they "are free for any use (attribution appreciated but not required)"

It would be better if they could be in CC-0, as this license includes a workaround for countries where public domain does not exist.

Reply:
Additional information: just keep in mind that open source licenses are used to give right to the users and protect them from when they use the software (use, study, modify, redistribute)

If there is nothing explained in a package regarding the protection of users and what they can do and not do, then usually, both FSF and OSI recommendation is to avoid it.